hazelcast/src/main/java/com/hazelcast/core/Member.java

```diff
@@ -52,9 +52,16 @@
     InetSocketAddress getInetSocketAddress();
 
     /**
+     * Get the capabilities that this member has in the cluster
+     * @return the capabilities for this member.
+     * @since 3.5.2-atlassian-25
+     */
+    Set<Capability> getCapabilities();
+
+    /**
      * Change the capabilities this member has in the cluster
      * @param capabilities The new capabilities for this member.
-     * @since 3.4
+     * @since 3.3.2-atlassian-5
```

Comment:
Unless and until the Capabilities change is accepted by upstream hazelcast/hazelcast, I put the `atlassian` version in here...